cogs/tld.py

```diff
@@ -0,0 +1,63 @@
+import random
+
+import voxelbotutils as utils
+
+
+class TldCommands(utils.Cog):
+
+    TLD_URL = "https://data.iana.org/TLD/tlds-alpha-by-domain.txt"
+
+    def __init__(self, bot:utils.Bot):
+        super().__init__(bot)
+        self.tld_list = None
+
+    async def load_tlds(self):
+        """Loads tlds into the cache"""
+
+        async with self.bot.session.get(self.TLD_URL) as r:
+            data = await r.text()
+        self.tld_list = data.split("\n")
+
+
+    @utils.group(aliases=['tld'])
+    async def tlds(self, ctx):
+        """
+        Allows some functions on tlds
+        """
+
+        pass
```

Comment:
This should send help to the user by default, if the command itself does nothing.